### PR TITLE
feat: create index for path base search

### PR DESCRIPTION
**Pull request by farnabaz**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/module.ts [hunk 1]
```diff
@@ -262,6 +262,11 @@ export default defineNuxtModule<ModuleOptions>({
           route: `/api/${options.base}/query`,
           handler: resolveRuntimeModule('./server/api/query')
         },
+        {
+          method: 'get',
+          route: `/api/${options.base}/find_one/:qid/**:slug`,
```
Comment (Atinux) [inline src/module.ts:267]: Could you confirm it works with slug being without any `/` @farnabaz ?

Trying locally on Studio and seems that it sends back to me a 405

Comment (Atinux) [reply src/module.ts:267]: Might be worth adding tests into radix3 https://github.com/unjs/radix3/blob/main/tests/insert.test.ts

Comment (pi0) [reply src/module.ts:267]: https://github.com/unjs/radix3/issues/8

Comment (farnabaz) [reply src/module.ts:267]: @Atinux I'm going to remove this endpoint and integrate Index with `/query` endpoint

**Pull request merged**